### PR TITLE
feat: Add cheqd chain

### PR DESCRIPTION
**Pull request by ankurdotb**
This update adds a PR to add cheqd-mainnet-1 (chain name: cheqd)

### CODE REVIEW

File: cosmos/cheqd.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  "chainName": "cheqd",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/cheqd/chain.png",
+  "rpc": "https://rpc.cheqd.net",
+  "rest": "https://api.cheqd.net",
+  "nodeProvider": {
+    "name": "cheqd Network",
+    "email": "security-github@cheqd.io",
+    "website":"https://cheqd.io/"
+  },
+  "walletUrl": "https://cheqd.omniflix.co",
```
Comment (HeesungB) [inline cosmos/cheqd.json:12]: @ankurdotb Could you remove this field?

Comment (HeesungB) [reply cosmos/cheqd.json:12]: @Tweeddalex yes!

**Comment by Tweeddalex** [comment]
Hey @HeesungB I think this should be ready to merge, unless there's any other changes :)

**Comment by HeesungB** [comment]
@ankurdotb @Tweeddalex 
Chain identifier unmatched: (expected: cheqd, actual: cheqd-mainnet)
Your `Chain identifier` is `cheqd-mainnet`, So Could you change your file, image file directory name to `cheqd-mainnet.json`, `images/cheqd-mainnet`?

**Comment by Tweeddalex** [comment]
Hey @HeesungB I've now renamed the file and folder as you suggested

File: cosmos/cheqd-mainnet.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{
+  "chainId": "cheqd-mainnet-1",
+  "chainName": "cheqd",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/cheqd/chain.png",
```
Comment (HeesungB) [inline cosmos/cheqd-mainnet.json:4]: Could you change this link to `https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/cheqd-mainnet/chain.png`

Comment (HeesungB) [reply cosmos/cheqd-mainnet.json:4]: cheqd -> cheqd-mainnet

Comment (Tweeddalex) [reply cosmos/cheqd-mainnet.json]: Ah yep, sorry for the inconvenience!

**Comment by HeesungB** [comment]
@Tweeddalex  Chain logos should be in PNG format in 256x256px resolution.
Could you resize image?

**Comment by Tweeddalex** [comment]
@HeesungB updated the image size - lets try one more time, fingers crossed!

**Comment by HeesungB** [comment]
@ankurdotb @Tweeddalex Thank you for contribution

**Pull request merged**